javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinterTest.java

```diff
@@ -803,7 +803,7 @@ void addASecondStatementToExistingMethod() throws IOException {
         assertEquals("public void someMethod() {" + EOL
                 + "        String test = \"\";" + EOL
                 + "        String test2 = \"\";" + EOL
-                + "}", LexicalPreservingPrinter.print(methodDeclaration));
+                + "    }", LexicalPreservingPrinter.print(methodDeclaration));
```

Comment:
Mmm, I think that ideally the right brace should not be intended because the opening method was not indented. However that would be really tricky to implement as we should remember where the brace was opened and what indentation we had there

Reply:
What should we do about it?

Reply:
@ftomassetti - what do you think?

Reply:
We could leave it as it is for now, perhaps adding a comment in the test because if later someone implement the complex but more correct behavior he will break this test

Reply:
Okay, @moraispgsi, let's get that comment in and merge this thing.

Reply:
@matozoid I added the comment.